src/backer/ogmiosing.py

```diff
@@ -0,0 +1,72 @@
+# -*- encoding: utf-8 -*-
+"""
+CARDANO-BACKER
+backer.ogmiosing module
+
+class to support subcribe tip from ogmios and cardano node
+"""
+import ogmios
+from hio.base import doing
+from keri import help
+from ogmios.statequery import QueryNetworkTip
+
+logger = help.ogler.getLogger()
+
+
+class Ogmioser(doing.DoDoer):
+
+    def __init__(self, on_tip=False, tip_slot=0, **kwa):
+        self.on_tip = on_tip
+        self.tip_slot = tip_slot
+        doers = [doing.doify(self.queryTipDo), doing.doify(self.crawlBlockDo)]
+        super(Ogmioser, self).__init__(doers=doers, **kwa)
+
+    def crawlBlockDo(self, tymth=None, tock=0.0):
+        self.wind(tymth)
+        self.tock = tock
+        _ = (yield self.tock)
+
+        try:
+            with ogmios.Client() as client:
+                _, tip, _ = client.find_intersection.execute([ogmios.Origin()])
+                _, _, _ = client.find_intersection.execute([tip.to_point()])
+
+                while True:
+                    direction, tip, point, _ = client.next_block.execute()
+                    if direction and direction == ogmios.Direction.forward:
+                        slot = point.slot
+
+                        if self.tip_slot > 0 and self.tip_slot <= slot:
+                            self.on_tip = True
+                            logger.debug(
+                                f"Reached tip {self.tip_slot} at slot {slot}")
+                            return self.tock

```

Comment:
I think this should still be yield, so that we continue to sync when at the tip (new blocks will come).

We could increase `self.tock` once we've gotten to the tip to reduce CPU usage and not overload ogmios too.

Reply:
sounds good to me - so tock is `0.0` while syncing and once at the tip it goes to `1.0`